zombies.js

```diff
@@ -78,137 +94,210 @@
  * @property {method} getPack              Returns private variable `pack`.
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
+function Player(name, health, strength, speed) {
+  this.name = name;
+  this.health = health;
+  this.strength = strength;
+  this.speed = speed;
+  this.isAlive = true;
+  this.equipped = false;
```

Comment:
I think it would be a bit clearer if your defined your `let` variables above your `this.properties`.